plugins/flytekit-pydantic/README.md

```diff
@@ -0,0 +1,29 @@
+# Flytekit Pydantic Plugin
+
+Pydantic is a data validation and settings management library that uses Python type annotations to enforce type hints at runtime and provide user-friendly errors when data is invalid. Pydantic models are classes that inherit from `pydantic.BaseModel` and are used to define the structure and validation of data using Python type annotations.
+
+The plugin adds type support for pydantic models.
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-pydantic
+```
+
+
+## Type Example
+```python
+from pydantic import BaseModel
+import flytekitplugins.pydantic # This import will enable you to add FlyteFiles and FlyteDirectories to you BaseModels
```

Comment:
we don't need it. flytekit will auto load the transformer if the plugin is installed.